spec/spec_helper.rb

```diff
@@ -4,18 +4,19 @@
 
 require (Pathname.new(__FILE__).dirname + "../lib/suspenders").expand_path
 
-Dir["./spec/support/**/*.rb"].sort.each { |file| require file }
+Dir["./spec/support/*.rb"].sort.each { |file| require file }
```

Comment:
The `spec/support` folder now has the following structure:

- `spec/support/*` - Normal spec support files
- `spec/support/fixtures` - Fake project template directory and other fixtures
- `spec/support/fakes` - Binary stubs and Ruby file stubs